tests/lookup/tests.py

```diff
@@ -127,6 +129,17 @@ def test_in_bulk(self):
         with self.assertRaises(TypeError):
             Article.objects.in_bulk(headline__startswith='Blah')
 
+    def test_in_bulk_lots_of_ids(self):
+        TEST_RANGE = 2000
+        max_query_params = connection.features.max_query_params
+        expected_num_queries = ceil(TEST_RANGE / max_query_params) if max_query_params else 1
+
+        Author.objects.all().delete()
```

Comment:
It might be a tiny bit faster to create `TEST_RANGE - Author.objects.count()` objects rather than deleting the existing ones.